components/scream/src/control/atmosphere_driver.cpp

```diff
@@ -147,11 +147,6 @@ void AtmosphereDriver::initialize (const ekat::Comm& atm_comm,
   // Initialize atm inputs
   init_atm_inputs ();
 
-  // Now we can inspect the dag, including also checking that
-  // all fields are correctly inited. Any unmet dependency in
-  // the dag at this point has to be treated as an error.
-  inspect_atm_dag ();
```

Comment:
It is, but not here. The method `inspect_atm_dag` is public, and it can be used from outside. In case the SC is on (i.e., for CIME runs) it is called [here](https://github.com/E3SM-Project/scream/blob/7e806b8efcc7f325f631493b2c760efb8760d1e4/components/scream/src/mct_coupling/scream_cxx_f90_interface.cpp#L164).